python/MeshData.sip

```diff
@@ -24,12 +24,12 @@ class MeshData
 public:
     MeshData();
     virtual ~MeshData();
-    PyObject* getFacesAsBytes();
-    PyObject* getVerticesAsBytes();
+    std::vector<char> getVerticesAsBytes();
```

Comment:
Can we make this an uint8_t instead? Maybe even add a "typedef bytearray std::vector<uint8_t> so the code becomes a bit more self-explanatory.

(Also applies below.)